Dockerfile

```diff
@@ -0,0 +1,25 @@
+ARG ROS_DISTRO=galactic
+
+FROM ros:$ROS_DISTRO-ros-base
+
+# Use bash instead of sh
+SHELL ["/bin/bash", "-c"]
+
+WORKDIR /ros_ws
```

Comment:
Path to `colcon` workspace is `/ros_ws` and not `ros2_ws`.

Reply:
:heavy_check_mark: